modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -29,6 +29,9 @@ class OrtResultAnalyzer : ManualAnalyzer() {
         private val LOGGER = LoggerFactory.getLogger(OrtResultAnalyzer::class.java)
     }
 
+    private val disableScopeReductionProperty = "disable.scope.reduction"
```

Comment:
So, if you want to invert the logic here, how about simply calling this "keep excluded artifacts"?